.storybook/themes/commerce.scss

```diff
@@ -3,43 +3,577 @@
+					components: (
+						stack: (
+							gap: var(--global-spacing-1),
+						),
+					),
+				),
+				category-carousel-slide-title: (
+					font-size: var(--global-font-size-2),
+				),
+				hero-stacked: (
+					gap: var(--global-spacing-4),
+				),
+				hero: (
+					color: var(--global-neutral-10),
+					components: (
+						button: (
+							text-decoration: none,
+						),
+						stack: (
+							gap: var(--global-spacing-3),
+						),
+						heading: (
+							font-size: var(--global-font-size-11),
+							line-height: var(--global-line-height-7),
+						),
+						paragraph: (
+							font-size: var(--global-font-size-3),
+							line-height: var(--global-line-height-3),
+						),
+					),
+				),
+				hero-light: (
+					color: var(--global-white),
+				),
+				hero-dark: (
+					color: var(--global-neutral-10),
+				),
+				hero-image: (
+					max-height: 65vh,
 					width: 100%,
 				),
-				stack-divider-horizontal: (
-					border: none,
-					border-left-width: 2px,
-					border-left-style: solid,
-					border-left-color: #712d2d,
+				hero-image-stacked: (
 					height: auto,
-					width: 1px,
+					width: 100%,
+				),
+				hero-text-left: (
+					max-width: 100%,
+					justify-self: flex-start,
+					padding: 0 var(--global-spacing-3),
+				),
+				hero-text-stacked: (
+					max-width: 24rem,
 				),
-				carousel-image-counter: (
-					color: #191919,
+				hero-subheadline: (
+					components: (
+						heading: (
+							font-size: var(--global-font-size-7),
+							font-weight: var(--global-font-weight-4),
+							line-height: var(--global-line-height-4),
+						),
+					),
 				),
-				carousel-fullscreen-image-counter: (
-					color: white,
+				quilted-image: (
+					margin: 0 auto,
+					max-width: calc(var(--layout-content-max-width) * 1px),
+					components: (
+						heading: (
+							font-size: var(--global-font-size-7),
+							font-weight: var(--global-font-weight-7),
+							line-height: var(--global-line-height-4),
+							margin-bottom: var(--global-spacing-2),
+							margin-left: var(--global-spacing-3),
+						),
+					),
 				),
-				carousel-additional-controls: (
-					display: none,
+				quilted-image__wrapper: (
+					gap: var(--global-spacing-5),
+					grid-template-areas: "top" "middle" "bottom",
+					grid-auto-flow: "column",
 				),
+				quilted-image__overlay-text--light: (
+					color: var(--global-white),
+				),
+				quilted-image__overlay-text--dark: (
+					color: var(--global-neutral-10),
+				),
+				quilted-image__overlay: (
+					left: var(--global-spacing-3),
+					bottom: var(--global-spacing-3),
+					padding-right: var(--global-spacing-3),
+					components: (
+						paragraph: (
+							font-weight: var(--global-font-weight-7),
+							line-height: var(--global-line-height-4),
+							font-size: var(--global-font-size-5),
+						),
+						stack: (
+							gap: var(--global-spacing-3),
+							align-items: start,
+						),
+					),
+				)
 			),
 		),
 		desktop: (
+			alias: (),
```

Comment:
this alias can be removed for desktop as it's empty. Probably a move for another day having breakpoint-dependent aliases. I suppose it's a style decision but it's not necessary when I removed it

Reply:
@JackHowa : Good call! Removed.